libs/hdf-converters/src/utils/fingerprinting.ts

```diff
@@ -60,7 +61,8 @@ const fileTypeFingerprints: Record<INPUT_TYPES, string[]> = {
   [INPUT_TYPES.NETSPARKER]: [],
   [INPUT_TYPES.SCOUTSUITE]: [],
   [INPUT_TYPES.NOT_FOUND]: [],
-  [INPUT_TYPES.VERACODE]: []
+  [INPUT_TYPES.VERACODE]: [],
+  [INPUT_TYPES.GOSEC]: []
```

Comment:
put the fields here you're using to uniquely identify with here since the source file is json so can parse it in one step

Reply:
added the fields to gosec

Reply:
make sure capitalization is consistent between what you put here and the file.

Reply:
changed GOSEC to GoSec

Reply:
changed as requested